tests/test_functorKeys.py

```diff
@@ -112,6 +112,52 @@ def testPointKey(self):
         self.doTestPointKey("D", lsst.afw.table.Point2DKey,
                             lsst.afw.geom.Point2D)
 
+    def doTestBoxKey(self, pointFieldType, functorKeyType, valueType):
+        schema = lsst.afw.table.Schema()
+        fKey0 = functorKeyType.addFields(schema, "a", "box", "pixel")
+        minKey = pointFieldType(schema["a_min"])
+        maxKey = pointFieldType(schema["a_max"])
+        # we create two equivalent functor keys, using the two different
+        # constructors
```

Comment:
don't have to do 80 chars, I don't think...

Reply:
Correct, this is just what my editor does when I tell it to wrap (since it treats comments like docs, which do have to by 80 chars, I think).  Anyhow, I agree it looks funny here; will fix.